docker-compose.yml

```diff
@@ -1,13 +1,19 @@
 version: "3.3"
+networks:
+  web:
+    external: true
 services:
   openvas:
-    ports:
-      - "8080:9392"
     environment:
       - "PASSWORD=Your admin password here"
+    dns:
```

Comment:
Same with this, by default docker copies `/etc/resolv.conf` from the host to the container. There's no need for this, unless you have a host configuration issue